File: src/server/BuildingBlock/MovieTicket.Domain/Entities/Transaction.cs
using MovieTicket.Core.Domain;

namespace MovieTicket.Domain.Entities;

public class Transaction : EntityBase
{
    public Guid? ReservationId { get; set; }
    public long Total { get; set; }
    public string Message { get; set; }
    public TransactionState TransactionState { get; set; }
}

File: src/client/mobile/app/src/main/java/com/superman/movieticket/domain/entities/Transaction.kt
package com.superman.movieticket.domain.entities

import com.google.gson.annotations.SerializedName
import com.superman.movieticket.core.entity.BaseEntity
import java.io.Serializable

data class Transaction (
    @SerializedName("ReservationId")

    val reservationId: String,
    @SerializedName("total")
    val total: Long
): BaseEntity(), Serializable